docs/README.md

```diff
@@ -23,7 +23,7 @@ Important files:
 - `conf.py`: defining settings
 - `Makefile` and `make.bat`: not exactly sure, we use defaults
 - `index.rst`: Define the index of the final documentation
-- `ktpbase`: empty ktpbase implementation so other python modules do not give an exception when imported.
+- `openstf_dbc`: empty openstf_dbc implementation so other python modules do not give an exception when imported.
```

Comment:
Is this true? I think this line can be removed completely?

Reply:
Good one, I removed the line